readme.md

```diff
@@ -82,6 +82,63 @@ const reducer = handleActions<State, ActionTypes, Actions>(
 export default reducer;
 ```
 
+### Type utils
+
+`safe-redux` also provides some type utils to work with Redux.
+
+#### `Payload`
+
+Get the type of the payload of the action created by an action creator.
+
+#### `BindAction`
+
+Changes the return type of an action creator to `void`. In the context of a
+component the only important part of an action is the types of it's arguments.
+We don't rely on the return type.
+
+```typescript
+// src/pages/MyPage/MyPage.container.ts
+
+import { connect } from 'react-redux';
+import { Payload, BindAction } from '@housinganywhere/safe-redux';
+
+import { Actions } from './actions';
+import MyPage from './MyPage';
+
+type IncPayload = Payload<typeof Actions.incBy>; // { by: number }
```

Comment:
[req] as we were talking, please update the sample to make it more concrete about how to use it here.

Reply:
I checked Moneta UI and whatever use case I had looks like we don't have it anymore because it's not used there. So for now I'll remove it. If it's need it can be added.